### PR TITLE
feat: documented invokable scripts within tasks functionality in the CLI

### PR DESCRIPTION
**Pull request by Xenonic**
This PR documents the new flags available to the `create` and `update` subcommands for tasks in the CLI tool. PRs implementing these new features are here: https://github.com/influxdata/influx-cli/pull/432, https://github.com/influxdata/influx-cli/pull/433.

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.3/reference/cli/influx/task/create.md [hunk 1]
```diff
@@ -20,15 +20,20 @@ influx task create [task literal] [flags]
 |:-----|:------------------|:----------------------------------------------------------------------|:----------:|:----------------------|
 | `-c` | `--active-config` | CLI configuration to use for command                                  | string     |                       |
 |      | `--configs-path`  | Path to `influx` CLI configurations (default `~/.influxdbv2/configs`) | string     | `INFLUX_CONFIGS_PATH` |
+| `-r` | `--cron`          | [Cloud only] Cron expression to define when the task should run       | string     |                       |
```
Comment (sanderson) [inline content/influxdb/v2.3/reference/cli/influx/task/create.md:23]:
```suggestion
| `-r` | `--cron`          | _(InfluxDB Cloud only)_ Cron expression to define when the task runs             | string     |                       |
```

Comment (sanderson) [reply content/influxdb/v2.3/reference/cli/influx/task/create.md:23]: Is this really cloud only or will this be supported in InfluxDB 2.4?

File: content/influxdb/v2.3/reference/cli/influx/task/create.md [hunk 2]
```diff
@@ -20,15 +20,20 @@ influx task create [task literal] [flags]
...
+| `-e` | `--every`         | [Cloud only] Interval at which the task runs                          | string     |                       |
```
Comment (sanderson) [inline content/influxdb/v2.3/reference/cli/influx/task/create.md:24]:
```suggestion
| `-e` | `--every`         | _(InfluxDB Cloud only)_ Interval at which the task runs                          | string     |                       |
```

File: content/influxdb/v2.3/reference/cli/influx/task/create.md [hunk 3]
```diff
@@ -20,15 +20,20 @@ influx task create [task literal] [flags]
...
 | `-f` | `--file`          | Path to Flux script file                                              | string     |                       |
 | `-h` | `--help`          | Help for the `create` command                                         |            |                       |
 |      | `--hide-headers`  | Hide table headers (default `false`)                                  |            | `INFLUX_HIDE_HEADERS` |
 |      | `--host`          | HTTP address of InfluxDB (default `http://localhost:8086`)            | string     | `INFLUX_HOST`         |
 |      | `--http-debug`    | Inspect communication with InfluxDB servers.                          | string     |                       |
 |      | `--json`          | Output data as JSON (default `false`)                                 |            | `INFLUX_OUTPUT_JSON`  |
+| `-n` | `--name`          | [Cloud only] Name of the task                                         | string     |                       |
```
Comment (sanderson) [inline content/influxdb/v2.3/reference/cli/influx/task/create.md:31]:
```suggestion
| `-n` | `--name`          | _(InfluxDB Cloud only)_ Task name                                         | string     |                       |
```

File: content/influxdb/v2.3/reference/cli/influx/task/create.md [hunk 4]
```diff
@@ -20,15 +20,20 @@ influx task create [task literal] [flags]
...
 | `-o` | `--org`           | Organization name (mutually exclusive with `--org-id`)                | string     | `INFLUX_ORG`          |
 |      | `--org-id`        | Organization ID (mutually exclusive with `--org`)                     | string     | `INFLUX_ORG_ID`       |
 |      | `--skip-verify`   | Skip TLS certificate verification                                     |            | `INFLUX_SKIP_VERIFY`  |
+|      | `--script-id`     | [Cloud only] Script ID that gets executed instead of Flux             | string     |                       |
+|      | `--script-params` | [Cloud only] JSON parameters for the script to be executed            | string     |                       |
```
Comment (sanderson) [inline content/influxdb/v2.3/reference/cli/influx/task/create.md:36]:
```suggestion
|      | `--script-id`     | _(InfluxDB Cloud only)_ Invokable script ID to execute             | string     |                       |
|      | `--script-params` | _(InfluxDB Cloud only)_ Invokable script JSON parameters            | string     |                       |
```

File: content/influxdb/v2.3/reference/cli/influx/task/create.md [hunk 5]
```diff
@@ -57,3 +62,8 @@ influx task create $FLUX_TASK
 ```sh
 influx task create --file /path/to/example-task.flux
 ```
+
+##### Create a task with a script
+```sh
+influx task create --name "my task" --every "10h" --script-id 0001234
+```
```
Comment (sanderson) [inline content/influxdb/v2.3/reference/cli/influx/task/create.md:69]:
~~~suggestion
##### Create a task using an invokable script
```sh
influx task create \
  --name "my task" \
  --every "10h" \
  --script-id 0001234
```
~~~

File: content/influxdb/v2.3/reference/cli/influx/task/update.md [hunk 6]
```diff
@@ -28,6 +28,8 @@ influx task update [task literal] [flags]
 | `-i` | `--id`            | ({{< req >}}) Task ID                                                 | string     |                       |
 |      | `--json`          | Output data as JSON (default `false`)                                 |            | `INFLUX_OUTPUT_JSON`  |
 |      | `--skip-verify`   | Skip TLS certificate verification                                     |            | `INFLUX_SKIP_VERIFY`  |
+|      | `--script-id`     | [Cloud only] Script ID that gets executed instead of Flux             | string     |                       |
+|      | `--script-params` | [Cloud only] JSON parameters for the script to be executed            | string     |                       |
```
Comment (sanderson) [inline content/influxdb/v2.3/reference/cli/influx/task/update.md:32]:
```suggestion
|      | `--script-id`     | _(InfluxDB Cloud only)_ Invokable script ID to execute             | string     |                       |
|      | `--script-params` | _(InfluxDB Cloud only)_ Invokable script JSON parameters            | string     |                       |
```

**Review by sanderson — CHANGES_REQUESTED** [review]
Thanks for this @Xenonic. A few notes:

- This branch needs to be based off the `influx-cli-2.4` branch. 
- These changes should be applied to `/content/influxdb/v2.4/`, not `.../v2.3`.
- Will the new flags only be supported in Cloud or will 2.4 rollout with support for these? If so, we can remove the "Cloud only" flags.

**Comment by Xenonic** [comment]
Sorry I totally broke my branch with a force push, addressed feedback and fixed the branch here: https://github.com/influxdata/docs-v2/pull/4295

**Pull request closed without merging**